app/models/piece.rb

```diff
@@ -19,4 +19,19 @@ def move_to!(row, col)
     raise "Out of bounds" if row < 0 || row > 7 || col < 0 || col > 7
     update(row: row, col: col)
   end
+
+  def capture_piece(row, col)
+    other_piece = game.piece_at(row, col)
+    return true if game.path_obstructed? && other_piece.user != user
```

Comment:
Currently `path_obstructed?` doesn't exist in the Game class - are you trying to check that there's a piece at the destination?  If so then this could just become:

```ruby
return true if other_piece && other_piece.user != user
```

However that will never allow captures, so you probably want to fail if the other piece exists and is the *same* user:

```ruby
fail 'You cannot capture your own piece' if other_piece && other_piece.user == user
```